src/balancereader/pom.xml

```diff
@@ -114,7 +114,7 @@
         <dependency>
           <groupId>com.google.guava</groupId>
           <artifactId>guava</artifactId>
-          <version>29.0-jre</version>
+          <version>30.0-jre</version>
```

Comment:
Any reason for going with 30.0 instead of 30.1.1?

Reply:
Nope! That was simply the latest major release. I've just updated it to get the `.1` minor release.